Tensile/KernelWriterAssembly.py

```diff
@@ -11880,15 +11784,15 @@ def vectorStaticRemainder(qReg, rReg, dReg, divisor, tmpVgpr, tmpSgpr):
     """
     shift = 32+1
     magic = ((2**shift) // divisor) + 1
-    kStr += inst("s_mov_b32", sgpr(tmpSgpr), hex(magic), "")
-    kStr += inst("v_mul_hi_u32", vgpr(tmpVgpr+1), vgpr(dReg), sgpr(tmpSgpr), "")
-    kStr += inst("v_mul_lo_u32", vgpr(tmpVgpr+0), vgpr(dReg), sgpr(tmpSgpr), "")
-    kStr += inst("s_mov_b32", sgpr(tmpSgpr), hex(shift), "")
-    kStr += inst("v_lshrrev_b64", vgpr(tmpVgpr,2), sgpr(tmpSgpr), vgpr(tmpVgpr,2), "")
-    kStr += inst("v_mov_b32", vgpr(qReg), vgpr(tmpVgpr), "vectorStaticDiv: quotient")
-    kStr += inst("s_mov_b32", sgpr(tmpSgpr), hex(divisor), "divisor")
-    kStr += inst("v_mul_lo_u32", vgpr(tmpVgpr), vgpr(qReg), sgpr(tmpSgpr), "vectorStaticDiv: product = quotient * divisor")
-    kStr += inst("_v_sub_co_u32", vgpr(rReg), "vcc", vgpr(dReg), vgpr(tmpVgpr), "vectorStaticDiv: remainder = dividend - product")
+    kStr += inst("s_mov_b32", sgpr(tmpSgpr), hex(magic), comment)
+    kStr += inst("v_mul_hi_u32", vgpr(tmpVgpr+1), vgpr(dReg), sgpr(tmpSgpr), comment)
+    kStr += inst("v_mul_lo_u32", vgpr(tmpVgpr+0), vgpr(dReg), sgpr(tmpSgpr), comment)
+    kStr += inst("s_mov_b32", sgpr(tmpSgpr), hex(shift), comment)
+    kStr += inst("v_lshrrev_b64", vgpr(tmpVgpr,2), sgpr(tmpSgpr), vgpr(tmpVgpr,2), comment)
+    kStr += inst("v_mov_b32", vgpr(qReg), vgpr(tmpVgpr), comment)
+    kStr += inst("s_mov_b32", sgpr(tmpSgpr), hex(divisor), comment)
+    kStr += inst("v_mul_lo_u32", vgpr(tmpVgpr), vgpr(qReg), sgpr(tmpSgpr), comment)
+    kStr += inst("_v_sub_co_u32", vgpr(rReg), "vcc", vgpr(dReg), vgpr(tmpVgpr), comment)
```

Comment:
Minor detail, but this probably doesn't need the same comment on every line. I'd probably just put the comment at the start and then something different to mark the end of the vectorStaticDiv?

Reply:
Hi Alex,

Sometime we will put /*......*/ at beginning of code block, ex: local read, which shows purpose of code block.
If we put /*...*/ at beginning of Reminder/Divide, it will confuse us with code block.
So I choose to add same comment in tail comment of instruction.
We know those comment is for divide and didn't break code block.
Does that work for you??

Reply:
Yea, that works. Whatever you think will be most readable, and whatever helps speed up debugging the generated kernels.